### DISCUSSION TITLE
Sniffnet has been selected for the GitHub Accelerator Program :octocat: 🚀

### DISCUSSION THREAD

**Discussion by GyulyVGC** [GyulyVGC/sniffnet#133] [Announcements]
<h3 align="center">Sniffnet has been selected for the GitHub Accelerator Program! <br><br>:octocat: 🚀</h3>

<p>
<a href="#x">
<img alt="" src="https://user-images.githubusercontent.com/100347457/227869349-cc588f0f-13bf-45e4-aaf6-b55a5c641191.png" width=100%>
</a>
</p>

<a href="https://github.com/GyulyVGC/sniffnet"><img align="right" alt="" src="https://github.com/GyulyVGC/sniffnet/blob/main/resources/repository/header_repository.png?raw=true" width="50%"/></a>

[Sniffnet](https://github.com/GyulyVGC/sniffnet) has **grown a lot** during the past months, and it’s been a pleasure for me to spend most of my spare time on its development.
I'm not gonna lie: passing hours and hours on this project has become my favorite hobby lately.
<br>I can’t deny that sometimes **it’s been hard** to balance Sniffnet’s development with the daily University routine, made of never-ending lectures, group projects, deliveries, and home study. 
I often ended up coding late (I mean _very_ late) at night or skipping meals/lectures, since 24 hours a day never seemed enough.
Seeing Sniffnet getting traction and evolving, improving day after day, motivated me to **keep pushing** despite the obstacles along the way.
<br>**Long story short**: having more time to dedicate to open-source was one of my primary wishes (and needs 😅).

<p>
<a href="#x">
<img alt="" src="https://user-images.githubusercontent.com/100347457/227869349-cc588f0f-13bf-45e4-aaf6-b55a5c641191.png" width=100%>
</a>
</p>

<a href="https://accelerator.github.com">
<img width="40%" alt="" align="right" src="https://user-images.githubusercontent.com/100347457/230674800-c4bf0556-c1de-4bc6-aef3-aab9dae5204f.png"/>
</a>

Today, the 12th of April 2023, it’s such a joy for me to announce that **Sniffnet has been selected for the GitHub Accelerator Program** 🎉
<br>The [GitHub Accelerator](https://accelerator.github.com) is a program to pioneer new ways for developers to sustainably work on their **open-source projects full-time**.
20 promising and influential open-source projects [have been elected](https://github.blog/2023-04-12-github-accelerator-our-first-cohort-and-whats-next/), out of more than a thousand total applicants.
The maintainers of such projects will participate in a 10-week program receiving an initial **sponsorship** as well as **mentorship** from open-source leaders and enterprise partners on how to grow software projects and build durable streams of funding.
<br>This means that during the next few months I’ll be able to **work on Sniffnet full-time**, with the aspiration to bring it to the next level.
I’ve lots of ideas for new features and improvements and I’m looking forward to implementing them all 😁

<p>
<a href="#x">
<img alt="" src="https://user-images.githubusercontent.com/100347457/227869349-cc588f0f-13bf-45e4-aaf6-b55a5c641191.png" width=100%>
</a>
</p>

<a href="#x">
<img alt="" align="right" src="" width=40%>
</a>

Turning such a huge passion into a full-time job is the best thing I could’ve ever asked for.
I truly wish that this journey will continue even after the 10 weeks of the program, with the hope of working on open-source for my **entire career**.
<br>I believe that more and more programmers deserve to have opportunities like this one, and I hope the Accelerator will pave the way to a **brighter perspective for the open-source community**.
<br>I can’t wait to see what the future has in store, and I feel blessed to have such an amazing occasion.
I’ll do my best to get the most out of this experience 🌟

<a href="#x">
<img alt="" src="https://user-images.githubusercontent.com/100347457/231460175-6a0263d8-66d3-46e2-be61-2b35b912df24.jpg" width=100%>
</a>

**Comment by Themercee** [comment]
Good job! Very happy for you :)

**Comment by Viatorus** [comment]
Well done! :)

**Comment by addy** [comment]
You're killing it man, keep it up!

**Comment by hecrj** [comment]
Congratulations! :partying_face:

**Reply by GyulyVGC** [reply]
🤝🙏🦀

**Comment by romancitodev** [comment]
Congrats!

**Comment by deikatsuo** [comment]
Congrats 😁

**Comment by ppvan** [comment]
Congrats!

**Comment by tomekit** [comment]
Congratulations ! 
Glad to see interest around solving open-source funding problems.

**Comment by safira-2007** [comment]
Congratulations. Hard work pays off. You deserve this.

**Comment by ifallisequal** [comment]
Congratulations!
